package.json

```diff
@@ -232,6 +232,7 @@
     ]
   },
   "docs": "https://www.chromatic.com/docs/cli",
+  "packageManager": "yarn@1.22.19+sha256.732620bac8b1690d507274f025f3c6cfdc3627a84d9642e38a07452cc00e0f2e",
```

Comment:
Is this intentional?

Reply:
Yeah, `lint-staged` was failing for me without it. Can pull out to another PR if you like.

Reply:
Nah just checking.